src/lightkurve/correctors/designmatrix.py

```diff
@@ -274,7 +278,7 @@ def pca(self, nterms=6):
         # produces more stable results.
         from fbpca import pca  # local import because not used elsewhere
 
-        new_values, _, _ = pca(self.values, nterms, n_iter=10)
+        new_values, _, _ = pca(self.values, nterms, n_iter)
```

Comment:
```suggestion
        new_values, _, _ = pca(self.values, nterms, n_iter=n_iter)
```

This needs to be a keyword argument, because `raw` is the next positional argument and that's not what we want!